frontend/public/components/notification-drawer.tsx

```diff
@@ -108,20 +111,37 @@ const getUpdateNotificationEntries = (
   isLoaded: boolean,
   updateData: ClusterUpdate[],
   isEditable: boolean,
+  newerChannel: string,
+  newerChannelVersion: string,
   toggleNotificationDrawer: () => void,
 ): React.ReactNode[] =>
   isLoaded && !_.isEmpty(updateData) && isEditable
     ? [
         <NotificationEntry
           actionPath="/settings/cluster"
           actionText="Update cluster"
-          key="cluster-udpate"
+          key="cluster-update"
           description={updateData[0].version || 'Unknown'}
           type={NotificationTypes.update}
           title="Cluster update available"
           toggleNotificationDrawer={toggleNotificationDrawer}
           targetPath="/settings/cluster"
         />,
+        newerChannel && (
+          <NotificationEntry
+            actionPath="/settings/cluster?showChannels"
+            actionText="Edit channel"
+            key="channel-update"
+            description={`OpenShift ${newerChannelVersion} is available. If you are
```

Comment:
We probably want to tweak the text here. I believe there is a gap between when the channel is first available and the new version actually GAs, so this message could be misleading. cc @wking @megan-hall

Reply:
Makes sense to me.  Maybe just `The ${newerChannel} channel is available.  If you are...`?

Reply:
Should the `title` still be `OpenShift ${newerChannelVersion} is available`?

Reply:
Probably this instead?  
<img width="455" alt="Screen Shot 2020-07-21 at 1 27 27 PM" src="https://user-images.githubusercontent.com/895728/88086822-f4ec3180-cb55-11ea-8fc2-1f193fc2ec30.png">

Reply:
gonna drop `is` in the title so it is parallel to the first

Reply:
<img width="453" alt="Screen Shot 2020-07-21 at 1 29 09 PM" src="https://user-images.githubusercontent.com/895728/88086970-2d8c0b00-cb56-11ea-89da-0cace9fedb7a.png">